sovrin_client/agent/walleted.py

```diff
@@ -278,6 +278,17 @@ def handleEndpointMessage(self, msg):
         if typ == ACCEPT_INVITE and link is None:
             localIdr = self.wallet.defaultId
         else:
+            # if accept invite is not the message type
```

Comment:
@khagesh , I am assuming, the fix which you provided will not allow ping if link is not created. But did you confirm from product if they are ok with this change that user should not be able to do ping before accepting link invitations?